README.md

```diff
@@ -268,6 +273,7 @@ Table of Contents
 - [**Music Player GO**](https://github.com/enricocid/Music-Player-GO) <sup>**[[F-Droid](https://f-droid.org/app/com.iven.musicplayergo)]**</sup>
 - [**Odeon**](https://github.com/thibseisel/android-odeon) <sup>**[[F-Droid](https://f-droid.org/app/fr.nihilus.music)]**</sup>
 - [**Metro**](https://github.com/MuntashirAkon/Metro) <sup>**[[F-Droid](https://f-droid.org/app/io.github.muntashirakon.Music)]**</sup>
+- [**Pulse Music**](https://github.com/HardcodeCoder/PulseMusic) <sup>**[[F-Droid](https://f-droid.org/en/packages/com.hardcodecoder.pulsemusic)]**</sup>
```

Comment:
Unfortunately Pulse Music is no longer developed, it has been removed therefore (see #154). :-(

Reply:
Oops 😅

Reply:
No problem :smile: